docs/DocGenerator/lib/IndexData.py

```diff
@@ -0,0 +1,81 @@
+"""
+brief: Wazuh DocGenerator config parser.
+copyright: Copyright (C) 2015-2021, Wazuh Inc.
+date: August 04, 2021
+license: This program is free software; you can redistribute it
+         and/or modify it under the terms of the GNU General Public
+         License (version 2) as published by the FSF - Free Software Foundation.
+"""
+
+import os
+import re
+import json
+import requests
+from elasticsearch import Elasticsearch, helpers
+
+class IndexData:
+    """
+    brief: Class that indexes the data from JSON files into ElasticSearch.
+    """
+    def __init__(self, index, config):
+        self.conf = config
+        self.path = self.conf.documentation_path
+        self.index = index
+        self.regex = ".*json"
+        self.files = self.get_files()
+        self.es = Elasticsearch()
+        self.output = []
+
+    def test_connection(self):
+        """
+        brief: It verifies with an HTTP request that an OK response is received from ElasticSearch.
+        """
+        try:
+            res = requests.get("http://localhost:9200/_cluster/health")
+            if res.status_code == 200:
+                return True
+        except Exception as e:
+            print(e)
+            return False
+
+    def get_files(self):
+        """
+        brief: Recursively finds all the files that match with the regex provided.
```

Comment:
I think the term **recursively** makes the reader think this method will be called recursively. I suggest explaining this like:
_Finds all the files inside the documentation path that matches with doc_file_regex_